src/main/java/org/opentripplanner/routing/edgetype/factory/GTFSPatternHopFactory.java

```diff
@@ -306,6 +306,10 @@ public int compare(IndexedLineSegment a, IndexedLineSegment b) {
 
     private GtfsStopContext context = new GtfsStopContext();
 
+    // the location types for transfers.txt
+    public int STOP_LOCATION_TYPE = 0;
```

Comment:
should `STOP_LOCATION_TYPE` and `PARENT_STATION_LOCATION_TYPE` be `static final`?